client/my-sites/plugins/plugin-meta/index.jsx

```diff
@@ -424,19 +426,39 @@ const PluginMeta = React.createClass( {
 		analytics.ga.recordEvent( 'Plugins', 'Clicked Update All Sites Plugin', 'Plugin Name', this.props.pluginSlug );
 	},
 
+	getExtensionPath( plugin ) {
+		const pluginSlug = get( plugin, 'slug' );
+		const sections = sectionsModule.get();
+		const index = findIndex( sections, { name: pluginSlug } );
+
+		if ( ( index === -1 ) || ! includes( sections[ index ].envId, config( 'env_id' ) ) ) {
+			return '';
+		}
+
+		return get( sections[ index ], 'paths', [] )[ 0 ];
+	},
+
 	render() {
 		const cardClasses = classNames( 'plugin-meta__information', {
 			'has-button': this.hasOrgInstallButton(),
 			'has-site': !! this.props.selectedSite,
 			'is-placeholder': !! this.props.isPlaceholder
 		} );
+		let target = '_blank';
 
 		const plugin = this.props.selectedSite && this.props.sites[ 0 ] ? this.props.sites[ 0 ].plugin : this.props.plugin;
 		let actionLinks = get( plugin, 'action_links' );
 		if ( get( plugin, 'active' ) && isEmpty( actionLinks ) ) {
 			actionLinks = this.getDefaultActionLinks( plugin );
 		}
 
+		const path = this.getExtensionPath( plugin );
+
+		if ( path && has( actionLinks, 'Settings' ) ) {
```

Comment:
@rickybanister Not sure it's as simple as just replacing the existing Settings link. For example, WP Job Manager has a _WP Admin_ link, while Sensei calls it _Configure_. I don't think we could just look at whether or not the URL contains `wp-admin` either, since there are some plugins with multiple buttons that could all link to WP Admin, so it won't really be possible to know which is the link for the settings.

What about introducing a new button instead of replacing the existing one? And in that case, what would we call it?